server/server/src/cors.rs

```diff
@@ -1,9 +1,9 @@
 use actix_cors::Cors;
 use actix_web::http::header;
-use service::settings::Settings;
+use service::settings::{is_develop, Settings};
 
 pub fn cors_policy(config_settings: &Settings) -> Cors {
-    let cors = if config_settings.server.develop && config_settings.server.debug_cors_permissive {
+    let cors = if is_develop() {
```

Comment:
as per pr comment, removed this as i didn't see any use cases, apart from automated tests (which we don't currently have)

Reply:
Use case is to run a dev build with the same settings as in production, e.g. when debugging a production issue with a debugger. In the worst case you may want to do run the debug build in production to be able to attach a debugger to the running process.

Reply:
Hmm i guess I was confused by the flag, would assume we would want to turn `on` cors in dev vs having it turned on all the time even in dev mode, i'll make an issue to reinstate it (but with explicit turn on cors in dev), to avoid further conflicts in this [pr](https://github.com/openmsupply/open-msupply/pull/49)

Reply:
https://github.com/openmsupply/open-msupply/issues/60